lego/apps/events/views.py

```diff
@@ -167,12 +168,8 @@ def administrate(self, request, *args, **kwargs):
         event_data = populate_event_registration_users_with_grade(event_data)
         return Response(event_data)
 
-    @decorators.action(
-        detail=True, methods=["POST"], serializer_class=StripeTokenSerializer
-    )
+    @decorators.action(detail=True, methods=["POST"], serializer_class=BaseSerializer)
     def payment(self, request, *args, **kwargs):
```

Comment:
Should we rename this to something more descriptive? Since it's no longer really a payment

Reply:
I couldn't really come up with anything better, and i feel like it makes sense, considering you have to post to the route in order to pay?